opts.go

```diff
@@ -271,3 +286,21 @@ func loadFromConfDir(c *libcni, max int) error {
 	c.networks = append(c.networks, networks...)
 	return nil
 }
+
+func checkPluginExists(c *libcni, confList *cnilibrary.NetworkConfigList) error {
+	for _, plug := range confList.Plugins {
+		plugin := plug.Network.Type
+
+		if !fileExistsInDir(c.pluginConfDir, plugin) {
+			return ErrCNIPluginNotFound
```

Comment:
```suggestion
			return fmt.Errorf("CNI plugin not found in %s/%s: %w", c.pluginConfDir, plugin, ErrInvalidConfig)
```

Reply:
thoughts? Not following the need to have a plurality of different error types for what amounts to incorrect cni config..

Reply:
My thought from the reader of the error message if I see the invalid cni config, I'm going to look at /etc/cni/net.d. If I see cni plugin not found, I'm looking at /opt/cni/bin. This helps reduce the troubleshooting time. The error message could be improved to give the reader the directories. I'm just thinking about it that way. Invalid config means something much different than cni plugin not found.